downstream/assemblies/platform/assembly-aap-containerized-installation.adoc

```diff
@@ -56,6 +56,7 @@ Each virtual machine (VM) has the following system requirements:
 | Disk IOPS   | 3000    
 |====
 
+If performing a bundled installation of the growth topology with `hub_seed_collections=true`, then 32 GB RAM is recommended. Note that with this configuration, install times can take 45 minutes or more to complete.
```

Comment:
What about instead of:

> install times can take 45 minutes or more to complete

We write something like:

> install time is going to increate and may take 45 or more minutes alone to complete seeding the collections.